meteor/client/ui/Shelf/AdLibPanel.tsx

```diff
@@ -238,23 +249,22 @@ const AdLibListView = withTranslation()(
 			}
 		}
 
-		renderRundownAdLibs() {
+		renderRundownAdLibs(uniquenessIds: Set<string>) {
 			const { t } = this.props
 
 			return (
 				<tbody className="adlib-panel__list-view__list__segment adlib-panel__list-view__item__rundown-baseline">
 					{this.props.rundownAdLibs &&
 						this.props.rundownAdLibs
-							.filter(
-								(item) =>
-									!item.isHidden &&
-									matchFilter(
-										item,
-										this.props.showStyleBase,
-										this.props.uiSegments,
-										this.props.filter,
-										this.props.searchFilter
-									)
+							.filter((item) =>
+								matchFilter(
```

Comment:
Why is the `.isHidden` is condition removed?

Reply:
I'll put that down to merge error/bad commit. I've restored that check https://github.com/olzzon/tv-automation-server-core/commit/92b4179541d94d3f7f67ab13353ef409f57af601